### PR TITLE
Ngwmn 1670 initial views

### PR DESCRIPTION
**Pull request by ayan-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+"""
+Application configuration settings
+
+"""
+DEBUG = True
```
Comment (mbucknell) [inline config.py]: I would prefer DEBUG default to False. It's safer if something where to go wrong with a prod deployment to fail back to DEBUG = False.

File: ngwmn/utils.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Utility functions
+
+"""
+from urllib.parse import urljoin
+
+from lxml import etree
+import requests as r
+
+
+def parse_xml(xml_string):
```
Comment (mbucknell) [inline ngwmn/utils.py]: Wd don't have to do anything now, but I think when we have a critical mass of "utils" to divide them up. Maybe xml_utils, fetch_utils, etc. But no need to do anything now.

File: ngwmn/utils.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    :rtype: etree._Element or None
+
+    """
+    lithography_target = urljoin(service_root, 'ngwmn/iddata')
+    query_params = {
+        'request': 'well_log',
+        'agency_cd': agency_cd,
+        'siteNo': location_id
+    }
+    resp = r.get(lithography_target, params=query_params)
```
Comment (mbucknell) [inline ngwmn/utils.py]: Probably want to do some error handling here.

File: ngwmn/views.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    :param str agency_cd: agency code for the agency that manages the location
+    :param location_id: the location's identifier
+
+    """
+    root = get_well_lithography(SERVICE_ROOT, agency_cd, location_id)
+    if root is not None and 'gml' in root.nsmap.keys():
+        geolocation_element = root.find('.//gml:Point/gml:pos', namespaces=root.nsmap)
+        geolocation = geolocation_element.text
+        latitude, longitude = geolocation.split(' ')
+        lat_lower = float(latitude) - 0.01
```
Comment (mbucknell) [inline ngwmn/views.py]: I bet this won't be the last time we need a bbox parameter. This could be a utils function.

File: ngwmn/views.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+            'CQL_FILTER': "((QW_SN_FLAG='1') OR (WL_SN_FLAG='1')) AND (BBOX(GEOM,{},{},{},{}))".format(
+                lon_lower,
+                lat_lower,
+                lon_upper,
+                lat_upper
+            )
+        }
+        params = {'request': 'GetFeature'}
+        target = urljoin(SERVICE_ROOT, 'ngwmn/geoserver/wfs')
+        resp = r.post(target, params=params, data=data)
```
Comment (mbucknell) [inline ngwmn/views.py]: Need to add error handling

File: config.py (hunk 1 above)
Comment (ayan-usgs) [reply config.py]: Ah... yes... I forgot to change that back to `False`.... :flushed:

File: ngwmn/views.py (hunk 4 above)
Comment (ayan-usgs) [reply ngwmn/views.py]: Sounds good.

File: ngwmn/views.py (hunk 5 above)
Comment (ayan-usgs) [reply ngwmn/views.py]: Added some simple error handling based on what WDFN UI does.

**Pull request merged**